testcases/misc_tests/cca_export_import_test.c

```diff
@@ -0,0 +1,1440 @@
+/*
```

Comment:
I guess we should have the standard file header here with copyright infos, license, etc.

Reply:
Ok, will do.